### ISSUE TITLE
Add page for imprint

### ISSUE THREAD

**Issue by TobiGr** [TeamNewPipe/website#4]
We need a page for an imprint. 
If we don't have one, we can get in trouble with German and European laws.

- [ ] Imprint information like address, contact and legal information
- [ ] Disclaimer for external links

Further, we should discuss whether we use cookies.

**Comment by theScrabi** [comment]
@TheAssassin about that Imprint, we have to discus that.

**Comment by TheAssassin** [comment]
Your justification is quite vague. I am going to clarify that.

First of all, let's regard German law: The first relevant one is the Telemediengesetz (TMG). Having a look at [§ 5 TMG](https://dejure.org/gesetze/TMG/5.html) which cares about the internet presence as a whole, an imprint is required for "geschäftsmäßige Online-Inhalte". As Newpipe and its website are all non-commercial, there is no need for an imprint due to TMG.
The other relevant law is the Rundfunkstaatsvertrag (RStV), more precisely [§ 55 RStV](http://www.dvtm.net/fileadmin/pdf/gesetze/13._RStV.pdf), which takes care of a website's contents. § 55 (1) is about "Telemedien", "die nicht ausschließlich persönlichen oder familiären Zwecken dienen" (that are not exclusively about personal or family purposes). As this software project is just a personal one for @theScrabi and the contributors, there is no need for an imprint (as long as there's no journalistic content like a blog, which would be dealt with by § 55 (2)).

About EU laws: Actually, there is no such thing like EU laws. There's the EU regulations that have to be implemented by national laws in a specified period of time (this depends on a regulation's type (yes, there's more than one, welcome to bureaucracy hell), if you are interested, you can read this [Wikipedia article](https://en.wikipedia.org/wiki/Regulation_(European_Union)). As a consequence, laws in the German jurisdiction are compatible to EU regulations, which implies that if you don't need an imprint for German laws, you won't need one for any EU regulations.

TL;DR: This website does not need an imprint in its current form (an informational, static page). As soon as this changes, we will have to rethink this decision.

About the disclaimer: I cannot count the times how often I read those, and thought "this is a waste of webspace". Many of them share the exact same wording. The bad thing about some of these wordings is: They are often illegitimate and thus ineffective. External links stuff is dealt with by (surprise) Telemediengesetz, more precise [§§ 7 ff. TMG](https://dejure.org/gesetze/TMG/7.html). This topic is even more complicated than the question whether you need an imprint or not (and if yes, what you have to put in it). You can read a summary [here (German)](http://blog-it-recht.de/2013/09/27/datenschutzklausel-und-haftungsausschluss-im-impressum/).

TL;DR: disclaimers are not necessary, because all they could do is inform the user about the contents of 
§§ 7 ff. TMG, which they could just look up themselves. It is better to not have one on your website than having a potentially illegal and abstruse wording that is in the end ineffective.

About the "use of cookies": I am not quite sure why this is a thing at the moment. The page is purely informational, and uses static HTML pages. There is no code that could even create cookies, or make use of them. Apart from that, this is a completely separate issue and should be discussed in a separate thread as soon as it gets relevant.

And last but not least the most important question: What about data privacy and the privacy notice things you can spot on may websites?
Well, surprise, TMG covers that part too. [§ 13 (1) TMG](https://dejure.org/gesetze/TMG/13.html) describes how a service provider has to inform their users about "Art, Umfang und Zwecke der Erhebung personenbezogener Daten" (type, amount and purpose of the collection of personal data). As the site does not collect any necessary data except for the webserver logs (which is verified to be legal), there is nothing to inform the user about.

**TL;DR: We don't need an imprint, we don't need a disclaimer, and we don't need a privacy note as long as the website remains a purely informational (and ideally static) website. As soon as this changes, we will have to discuss this issue again.**

@theScrabi next time, gimme a few minutes to write my wall of text.

Btw: apps [can be required](http://rechtsanwalt-schwenke.de/shocking-kaum-eine-mobile-app-hat-ein-zulaessiges-impressum/) to include an imprint, too. We should discuss that for Newpipe some day. (Although I'm quite sure that this is covered by Youtube's imprint as they are responsible for the content, perhaps we just need to include a link to Google's imprint.)

**Comment by TobiGr** [comment]
Perfect. Less work to do ;)
Adding an imprint and a disclaimer is probably a reflex, because I usually make wesites for companies and organisations.

**Comment by TheAssassin** [comment]
Tip for the future: Read the first part of [this blog post](http://blog-it-recht.de/2013/09/27/datenschutzklausel-und-haftungsausschluss-im-impressum/). You shouldn't add disclaimers in general. (There's just one exception: if that makes your bosses feel more comfortable, go add one).

**Comment by TobiGr** [comment]
Danke
